eo-parser/src/main/java/org/eolang/parser/EoSyntax.java

```diff
@@ -93,11 +99,13 @@ public final class EoSyntax implements Syntax {
                     "/org/eolang/parser/parse/expand-qqs.xsl",
                     "/org/eolang/parser/parse/expand-aliases.xsl",
                     "/org/eolang/parser/parse/resolve-aliases.xsl",
-                    "/org/eolang/parser/parse/add-default-package.xsl",
+                    "/org/eolang/parser/parse/add-default-package.xsl"
+                ).back(),
+                new TrDefault<>(new StHex()),
+                new TrClasspath<>(
                     "/org/eolang/parser/parse/explicit-data.xsl",
                     "/org/eolang/parser/parse/roll-bases.xsl"
-                ).back(),
-                new TrDefault<>(new StHex())
+                ).back()
```

Comment:
@volodya-lombrozo I also think we can try to remove `explicit-data.xsl` because we can generate the proper structure with data (with inner `<o>` element) everywhere right away, without relaying on future transformations. Let's add todo for it?

Reply:
@maxonfjvipon Sure, I've added it. Can you take a look one more time, please?